src/main/java/com/duckblade/osrs/dpscalc/calc/maxhit/magic/PoweredStaffMaxHitComputable.java

```diff
@@ -48,6 +54,7 @@ private interface StaffMaxHitProvider extends Computable<Integer>
 		.put(ItemID.CORRUPTED_STAFF_ATTUNED, ignored -> 31)
 		.put(ItemID.CRYSTAL_STAFF_PERFECTED, ignored -> 39)
 		.put(ItemID.CORRUPTED_STAFF_PERFECTED, ignored -> 39)
+    .put(ItemID.WARPED_SCEPTRE, WARPED)
```

Comment:
bad indent again lol
```suggestion
		.put(ItemID.WARPED_SCEPTRE, WARPED)
```